package.json

```diff
@@ -27,18 +27,19 @@
   },
   "homepage": "https://github.com/brenoc/eslint-config-vtex-react#readme",
   "dependencies": {
-    "eslint-config-prettier": "^4.2.0",
-    "eslint-config-vtex": "^10.0.0",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.0.0",
-    "eslint-plugin-react-hooks": "^1.0.0"
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-config-vtex": "^11.0.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react-hooks": "^1.6.1"
   },
   "devDependencies": {
-    "eslint": "^5.15.1",
-    "prettier": "^1.16.4",
-    "typescript": "^3.3.3333"
+    "eslint": "^6.0.1",
+    "prettier": "^1.18.2",
+    "typescript": "^3.5.3"
   },
   "peerDependencies": {
+    "eslint": "^6.0.1",
     "prettier": "^1.16.4",
     "typescript": "^3.3.3333"
```

Comment:
update typescript 👀

Reply:
it's ok to have the peer dependency with a lower version, so we don't force anyone to upgrade to the latest typescript version right away